contrib/babelfishpg_tsql/src/guc.c

```diff
@@ -1119,6 +1119,24 @@ define_custom_variables(void)
 							   GUC_NOT_IN_SAMPLE | GUC_DISALLOW_IN_FILE | GUC_DISALLOW_IN_AUTO_FILE,
 							   check_babelfish_dump_restore_min_oid, NULL, NULL);
 
+	DefineCustomIntVariable("babelfishpg_tsql.temp_oid_buffer_start",
+							gettext_noop("Internal. Specifies the start range of the buffer for temp oids"),
+							NULL,
+							&temp_oid_buffer_start,
```

Comment:
can we have int to uint32 conversion one time as part of GUC assignment itself. Convert incoming value into equivalent unit32 value. This will simply next oid logic.

Reply:
I think it is no different from having a #define, if I'm understanding you correctly? We can also save a little bit on function overhead

Reply:
I still think it will simply the logic on engine side. Right now it is more difficult to reason (and also error prone) with INT_MIN added at different places.